docs/messaging-service/index.md

```diff
@@ -0,0 +1,74 @@
+# Messaging Service
+
+## What is the Messaging service?
+
+The *Messaging service* is functionality within the *OneWelcome Identity Cloud* that enables you to configure and send notifications to your customers via SMS
+or email. The Messaging service serves as a notification gateway for other OneWelcome products to unify the configuration of the notification channels.
+
+The Messaging service can send out SMS and emails with the data directly embedded in the request, or can use templates from [*Customer
+Self-Styling*](../self-styling/index.md). Using templates gives you the option to adjust the look of the notifications.
```

Comment:
...to adjust the look and feel of notifications.

Reply:
fixed